deployment/ccip/changeset/internal/deploy_home_chain.go

```diff
@@ -28,7 +28,7 @@ const (
 
 	FirstBlockAge                           = 8 * time.Hour
 	RemoteGasPriceBatchWriteFrequency       = 30 * time.Minute
-	TokenPriceBatchWriteFrequency           = 3 * time.Second
+	TokenPriceBatchWriteFrequency           = 30 * time.Minute
```

Comment:
this is a big change just want to call it out to make sure its what you intended

Reply:
I was testing something locally and mistakenly changed it from 30 min to 3s in another PR. This is the intended one. :)